cookbooks/cdo-mysql/.kitchen.yml

```diff
@@ -1,14 +1,18 @@
 ---
 driver:
-  name: docker
-  use_sudo: false
+  name: dokken
+  privileged: true
+verifier:
+  name: inspec
 transport:
-  name: sftp
+  name: dokken
 provisioner:
-  name: chef_zero
-  require_chef_omnibus: 12.7.2
+  name: dokken
 platforms:
-  - name: ubuntu-14.04
+  - name: ubuntu
+    driver:
+      image: dokken/ubuntu-18.04
```

Comment:
This cookbook does not work with ubuntu 14 because the server package is no longer maintained, this is the very reason we're actively working on that upgrade. That said, I've manually tested the cookbook on an adhoc deployment (which currently uses ubuntu 14 with an aurora RDS instance) so it's been tested and verified working in that environment.